samples/.gitignore

```diff
@@ -0,0 +1 @@
+*/Properties
```

Comment:
Could probably revert this for now. We can always add it back later if it seems more definitely needed.